Competition/src/test/java/competition/subsystems/elevator/ElevatorSubsystemTest.java

```diff
@@ -0,0 +1,101 @@
+package competition.subsystems.elevator;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.math.BigDecimal;
+
+import org.junit.Test;
+import org.junit.experimental.theories.suppliers.TestedOn;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.elevator.ElevatorSubsystem.HatchLevel;
+import edu.wpi.first.wpilibj.MockDigitalInput;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+
+public class ElevatorSubsystemTest extends BaseCompetitionTest {
+
+    ElevatorSubsystem elevatorSubsystem;
+
+    @Override
+    public void setUp() {
+        super.setUp();
+        elevatorSubsystem = this.injector.getInstance(ElevatorSubsystem.class);
+    }
+
+    @Test
+    public void testElevatorConstructor() {
+        ElevatorSubsystem createElevatorSubsystem = this.injector.getInstance(ElevatorSubsystem.class);
+    }
+
+    @Test
+    public void testStopElevator() {
+        elevatorSubsystem.stop();
```

Comment:
This doesn't actually test that it stops. We need to make the elevator move, verify it's moving, then make it stop, and verify the stop.